generators/client/support/filter-entities.ts

```diff
@@ -25,7 +25,7 @@ import { type Entity } from '../../base-application/index.js';
 export const filterEntityPropertiesForClient = (entity: Entity): Entity => ({
   ...entity,
   fields: entity.fields.filter(field => !field.skipClient),
-  relationships: entity.relationships.filter(rel => !rel.skipClient && (rel.persistableRelationship || rel.relationshipEagerLoad)),
+  relationships: entity.relationships.filter(rel => !rel.skipClient),
```

Comment:
```suggestion
  relationships: entity.relationships.filter(rel => !rel.skipClient && (rel.persistableRelationship || rel.relationshipEagerLoad || rel.otherEntity?.jpaMetamodelFiltering)),
```

I think we should keep the filtering. Backreference is used in very few cases.

Reply:
Ok for me. I adjusted it according your suggestion and it works still works :-)